console/console-init/ui/src/Routes.tsx

```diff
@@ -97,7 +97,7 @@ export const AppRoutes = () => (
       getComponent={getDeviceDetailPage}
     />
     <LazyRoute
-      path="/iot-projects/:namespace/:projectname/create-device-form"
+      path="/iot-projects/:namespace/:projectname/devices/addform"
```

Comment:
addform name is not giving clarity which form we are adding. can we try something like addproject, adddevice, editdevice, clonedevice etc.